persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -188,16 +197,15 @@ public void execute() {
                 }
                 errStream.println("Created database `" + persistConfigurations.getDbConfig().getDatabase() + "`.");
 
-                String sqlFilePath = Paths.get(this.sourcePath, PERSIST_DIRECTORY,
-                                String.format(PersistToolsConstants.SQL_SCHEMA_FILE, entityModule.getModuleName()))
-                        .toAbsolutePath().toString();
+                String sqlFilePath = generatedSourceDirPath.resolve(String.format(PersistToolsConstants.SQL_SCHEMA_FILE,
+                                entityModule.getModuleName())).toAbsolutePath().toString();
                 try (Connection connection = getDBConnection(driver, persistConfigurations, true);
                      Reader fileReader = new BufferedReader(new FileReader(sqlFilePath,
                              StandardCharsets.UTF_8))) {
                     ScriptRunner sr = new ScriptRunner(connection);
                     sr.runScript(fileReader);
                 } catch (IOException e) {
-                    errStream.println("error occurred while reading SQL schema file, "
+                    errStream.println("Error occurred while reading SQL schema file, "
```

Comment:
```suggestion
                    errStream.println("ERROR: failed to read SQL schema file, "
```
Again do not append if possible, use printf()

Reply:
updated in [345097c](https://github.com/ballerina-platform/persist-tools/pull/140/commits/345097c9dba308883a3b916944e36e8ab3f03e74)